security/suricata/Makefile

```diff
@@ -3,6 +3,7 @@
 
 PORTNAME=	suricata
 PORTVERSION=	3.2.2
+PORTREVISION=	1
```

Comment:
You should bump PORTREVISION of `security/pfSense-pkg-suricata` instead of `security/suricata`. Can you change this please?